### ISSUE TITLE
Plugin not registerred

### ISSUE THREAD

**Issue by Scream4Life** [maxnowack/homebridge-dht-sensor#37]
I get this error message when I load homebridge

```
/usr/local/lib/node_modules/homebridge/lib/api.js:60
      throw new Error("The requested accessory '" + name + "' was not registered by any plugin.");
      ^

Error: The requested accessory 'DhtSensor' was not registered by any plugin.
    at API.accessory (/usr/local/lib/node_modules/homebridge/lib/api.js:60:13)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:254:42)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:81:38)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:40:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

My json file has this entry
 ```
  {
    "accessory": "DhtSensor",
    "name": "Sensor",
    "sensorType": 22,
    "gpioPin": 7
     },

```
and while installing your plugin (I tried with sudo and --unsafe-perm)
```
sudo npm install --unsafe-perm homebridge-dht-sensor
(node:2580) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
/home/pi
└── homebridge-dht-sensor@2.1.2 

npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN pi No description
npm WARN pi No repository field.
npm WARN pi No README data
npm WARN pi No license field.
```

DTH22 is in place and I installed the BCM2835 with this commands
```
sudo wget http://www.airspayce.com/mikem/bcm2835/bcm2835-1.50.tar.gz 
sudo tar zxvf bcm2835-1.50.tar.gz
cd bcm2835-1.50
./configure
make
sudo make check
sudo make install
cd ..
sudo rm -rf bcm2835*
```
I was not able to find the device-tree advanced option in the config screen.

Can you provide any help?

**Comment by maxnowack** [comment]
You're missing so `--global` parameter at the `npm install` command.

**Comment by Scream4Life** [comment]
I gave every command that I found and installed it without error
`sudo npm install --unsafe-perm -g homebridge-dht-sensor --global`

Unfortunately the plugin delivers this error after successfully load homebridge
```
Error: failed to read sensor
    at Error (native)
```
Problem on the modul or cables or would you guess something else?

**Comment by maxnowack** [comment]
sounds like you don't have connected the sensor to the GPIO pin you've configured. Checkout [pinout.xyz](https://pinout.xyz/) to get the correct number of the pin (BCM).

**Comment by Scream4Life** [comment]
![img_6830](https://cloud.githubusercontent.com/assets/8568430/23469854/25376426-fea5-11e6-83fd-9998ecc26614.JPG)

I tried a couple of changes with the same result. Actually I think my setup and the PIN is right. Nevertheless I tired 7 (wiring pi) and 4 as well. Maybe a broken sensor. Any other chance to read its data in order to check functionality?

**Comment by maxnowack** [comment]
You can try to use this library: https://github.com/technion/lol_dht22
If it's working, the wiring is correct.
In the picture, it looks like the DHT is connected to BCM17. In the config you posted, it's configured to use BCM7. Can this be the issue?

**Comment by Scream4Life** [comment]
BCM17 goes to the 433Mhz transmitter. It's #4 that goes to the data port of the DHT modul. I installed the lol_dh22 library (thanks for that!). After trying sudo ./loldht 4 (and 7...always confuse3d whether to take the BCM no or the WiringPi no) it says Data not good, skip. 

Maybe I got a bad DTH22 Modul :-(

Cables: I tried VCC with 3 and 5 (current) volt. Data goes to #4. A 10ohm piece between VCC and Data and ground from the - side on the right.
